html_css_proj/index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <style>
+    #navbar {
+      align-items:left;
+      position:fixed
+    }
+    @media only screen and (max-width: 800px) {
+  [class*="col-"] {
+    width: 100%;
+  }
```

Comment:
It would be super helpful if you always give an explanation for your requirements(Even on optional ones 👀). One of the many objectives code reviews have is to help the person being reviewed, and what better way to accomplish this than to give them a clear reason on why we are suggesting the changes 🤓 Kindly explain in your comments the reason for your changes/suggestions 😎